### PR TITLE
Fix/make labelling descriptive

### PR DESCRIPTION
**Pull request by NoelleDL**
### What

Changed label from "Contains" to "Search keywords". 

### How to review

"Search keywords" should now be the text for the label for the search input field. 

### Who can review

Anyone but me.

### CODE REVIEW

File: src/main/web/templates/html/t9-1.html [hunk 1]
```diff
@@ -305,7 +305,7 @@ <h3 class="flush float-left">Refine results</h3>
 						</div>
 						<div class="box__content">
 							<label class="field flush field--spaced field--text">
-								<span class="panel panel--separated--bottom block">Contains</span>
```
Comment (tgandrews) [inline src/main/web/templates/html/t9-1.html:308]: I don't think this file is used @jondewijones @rav-pradhan

File: src/main/web/templates/html/t9-2.html [hunk 2]
```diff
@@ -309,7 +309,7 @@ <h3 class="flush float-left">Refine results</h3>
 						</div>
```
Comment (tgandrews) [inline src/main/web/templates/html/t9-2.html:309]: Or this one.

File: src/main/web/templates/html/t9-1.html (hunk 1 above)
Comment (NoelleDL) [reply src/main/web/templates/html/t9-1.html:308]: I wasn't sure so I changed it to be safe. Should I revert this?

Comment (tgandrews) [reply src/main/web/templates/html/t9-1.html:308]: I think so. Ask Jon or Ravi to confirm but I don't think these are used. Maybe we can delete them?

Comment (rav-pradhan) [reply src/main/web/templates/html/t9-1.html:308]: Hey, so from my understanding the html files aren’t used and it’s only handlebars files we need to update when making any template changes; it should be fine, then, to revert the changes in those. With regards to deletion, I’m not sure if there’s a reason that they’re still in the codebase (Jon perhaps best placed to answer that)

**Pull request merged**